src/theme.js

```diff
@@ -3,6 +3,12 @@ export const theme = {
     white: "#ffffff",
     whisper: "#f5f5fa",
     mystic: "#e4e6e0",
+    waterloo: "#7E839A",
+    codGray: "#181818",
     black: "#000000",
   },
+  breakpoint: {
+    tablet: 1023,
+    phone: 590,
```

Comment:
I made phone breakpoint at 720px so it fits 2 MovieTiles in MoviesList. But I think it's fine for now